nemo_text_processing/inverse_text_normalization/de/taggers/whitelist.py

```diff
@@ -24,11 +24,16 @@ class WhiteListFst(GraphFst):
         e.g. misses -> tokens { name: "Mrs." }
     Args:
         tn_whitelist_tagger: TN whitelist tagger
+        input_file: path to a file with whitelist replacements
```

Comment:
could you add format of whitelist here, written\tnormalized

Reply:
done